src/custom/config.tsx

```diff
@@ -122,7 +123,13 @@ const PAGES: { [pageName: string]: Page } = {
     path: "/challenges",
     label: "Challenges",
     visible: (user, online) => true,
```

Comment:
Do we want this to be

```suggestion
    visible: (user, online) => config.ENABLE_CHALLENGES,
```

Or do we have a placeholder ready?

Reply:
For the purposes of us keeping this in development mode, this feature is already hidden in the sidebar. 

However, I assume we'll want to change this though to at least be `(user, online) => !online` since we don't cache challenge data locally the page would be redundant (non-authed users should be okay to view though)